static/app/components/events/highlights/highlightsDataSection.tsx

```diff
@@ -165,7 +170,11 @@ function HighlightsData({
     []
   );
 
-  const highlightTagItems = getHighlightTagData({event, highlightTags});
+  const highlightTagItems = getHighlightTagData({
```

Comment:
this removes it from the UI if the user has previously pinned `replayId`

Reply:
I this approach works to avoid people clicking dead links to replays but these highlights are set by users which means we'll be going against their wishes for displaying the link at the top of the page. Looking at the DB, there's about 26 projects in SaaS which have set `replayId` or `replay_id` to appear in their highlights.

Is there any harm in retaining these links for highlights? I think it might make more sense to figure out a way to prevent linking to missing replays and instead display some mention that the replay wasn't found rather than omitting it entirely.

Reply:
cc @jas-kas

Reply:
> Is there any harm in retaining these links for highlights? I think it might make more sense to figure out a way to prevent linking to missing replays and instead display some mention that the replay wasn't found rather than omitting it entirely.

I'm okay with retaining these links in Issue Details highlights. Are you suggesting to keep them there _**if**_ the user has pinned `replay_id` or in general for Issue Details highlights (for all users)? I'm good with either case but want to confirm the behaviour.

>  I think it might make more sense to figure out a way to prevent linking to missing replays **and instead display some mention that the replay wasn't found rather than omitting it entirely.**

This would be ideal (in the world where a user has explicitly pinned `replay_id`) -- that's what we do in the Replay section inside Issue Details as you know. :)

Reply:
addressed in https://github.com/getsentry/sentry/pull/76619